top_section_styles.css

```diff
@@ -0,0 +1,254 @@
+@import url(fonts.css);
+
+:root {
+    --primary: #3077C6;
+    --primary-dark: #10367C;
+    --secondary-yellow: #F5BD41;
+    --secondary-dark-mode1: #121212;
+    --secondary-dark-mode2: #1D1E1F;
+    --secondary-dark-mode3: #4D4D4D;
+    --primary-stroke: #F3F3F4;
+    --primary-text: #383838;
+    --primary-white: #FFFFFF;
+    --secondary-text: #BFBFBF;
+    --primary-bg: #EAF0F9;
+}
+
+body {
+    margin: 0;
+}
+.big-container {
+    width: 1440px;
```

Comment:
@IharDorinel use max-width here. My screen has other width I see the problem 
![image](https://user-images.githubusercontent.com/57193802/111909200-61188f80-8a6d-11eb-944e-42973ff395a3.png)